lib/mapper.js

```diff
@@ -108,6 +113,53 @@ function formatCompany(company){
   return ret;
 }
 
+ /**
+ * Formats location data for use with intercom
+ *
+ * https://doc.intercom.io/api/#user-model
+ *
+ * @param {Object} location
+ * @return {Object}
+ * @api private
+ */
+
+function formatLocation(context){
+  var location = context.location || {};
+  var codes = getCountryCodes(location.country);
+
+  var ret = {
+    type: 'location_data',
+    city_name: location.city || null,
+    continent_code: codes.continent || null,
+    country_code: codes.country || null,
+    country_name: location.country || null,
+    latitude: location.latitude || null,
+    longitude: location.longitude || null,
+    region_name: location.region || null,
+    postal_code: location.postalCode || null,
+    timezone: context.timezone || null
+  };
+
+  return ret;
+}
+
+/**
+ * Gets country code.
+ *
+ * @param {Object} name
+ * @return {Object} 
+ * @api private
+ */
+
+function getCountryCodes(name){
```

Comment:
This is called `getCountryCodes` but it actually gets a country code and a continent code. `getCountryAndContinentCodes` is a more descriptive name for it, but a more expressive way to do this would be to split it into two separate functions: `getCountryCode` and `getContinentCode`. That also has the side effect of making each easier to test.